notebook/static/tree/js/notebooklist.js

```diff
@@ -885,7 +885,7 @@ define([
 
         var item_path = that.selected[0].path;
 
-        window.open(utils.url_path_join('/files', item_path) + '?download=1');
+        window.open(utils.url_path_join(that.base_url, '/files', item_path) + '?download=1');
```

Comment:
The `/` should be removed, too, so this should be `, 'files'` instead of `, '/files'`